petejank/simple-pull-request-template
# Add remove button to dashboard entries

[DB-1452](http://url.pointing.to.the.ticket)

## Description

Added remove button to each of the dashboard entries. Clicking the remove button will permanently remove the item from the storage and reflect the change in the UI.

This PR also covers following changes:
- removed unused `lodash` dependency
- added generic `Button` component

## Screenshots

Include screenshots that visually describe the change in the table below:

| Before       | After       |
|--------------|-------------|
|![image](https://user-images.githubusercontent.com/4282713/66008882-8c45a280-e4b8-11e9-9cd1-fd21ff1f10c8.png)|![image](https://user-images.githubusercontent.com/4282713/66008840-759f4b80-e4b8-11e9-9e77-f77e6b203420.png)|

## How to test

1. Access http://localhost:8081/dashboard
2. Click any of the remove buttons
3. Check if selected dashboard entry was removed from the UI

## Pre-review checklist

- [x] Tested my change manually
- [x] Commented my code, particularly in hard-to-understand areas

## Post-review checklist

- [ ] Addressed all concerns raised during the review
- [ ] Received at least one approval

## Pre-merge checklist

- [ ] My branch is rebased with the latest `master`
